drivers/timer/nrf_rtc_timer.c

```diff
@@ -462,13 +480,26 @@ void rtc1_nrf5_isr(void *arg)
 #endif
 
 #ifdef CONFIG_TICKLESS_KERNEL
+	if (!expected_sys_ticks) {
+		if (_sys_clock_always_on) {
+			_sys_clock_tick_count = _get_elapsed_clock_time();
```

Comment:
_sys_clock_tick_count is already updated in program_max_cycles().

Reply:
will fix it.